src/main/java/org/folio/circulation/services/LostItemFeeRefundService.java

```diff
@@ -129,5 +115,25 @@ private ReferenceDataContext withLostItemPolicy(LostItemPolicy lostItemPolicy) {
       this.lostItemPolicy = lostItemPolicy;
       return this;
     }
+
+    private Collection<Account> getAccountsNeedRefund() {
```

Comment:
Does my suggestion read more like natural language to you?

```suggestion
    private Collection<Account> getAccountsNeedingRefunds() {
```

Reply:
What do you think to removing the `get` at the beginning? Personally I'm not a fan of this prefix a lot of the time.

Reply:
Agreed, removed the `get` prefix.